lang/python/python3-django-cors-headers/Makefile

```
@@ -0,0 +1,35 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=django-cors-headers
+PKG_VERSION:=3.1.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=django-cors-headers
+PKG_HASH:=5762ec9c2d59f38c76828dc1d4308baca4bc0d3e1d6f217683e7a24a1c4611a3
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-django-cors-headers
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Django application for handling the server headers required for Cross-Origin Resource Sharing (CORS).
+  URL:=https://github.com/ottoyiu/django-cors-headers
+  DEPENDS:=python3-django +python3-sqlparse
```

```suggestion
  DEPENDS:=+python3-light +python3-urllib python3-django
```

Looking at `import` lines (as described [here](https://github.com/openwrt/packages/pull/10376#discussion_r341376378)), there are [two](https://github.com/adamchainz/django-cors-headers/blob/3.1.1/corsheaders/checks.py#L4) [imports](https://github.com/adamchainz/django-cors-headers/blob/3.1.1/corsheaders/middleware.py#L2) from `urllib`.

Also, from [setup.py](https://github.com/adamchainz/django-cors-headers/blob/3.1.1/setup.py#L43), it doesn't look like django-cors-headers depends on sqlparse.